docs/license_en.md

```
@@ -1,83 +1,79 @@
-- Applies to the **latest version** at the time of license acquisition. Major updates may require upgrade fees.
-- If commercial use is confirmed after beginning use for non-profit personal, educational, or non-profit purposes, commercial license fees will be charged from the start of commercial use.
-- **Demonstrations at exhibitions and events** aimed at increasing project awareness are exempt from commercial licensing requirements.
+- All commercial licenses are **one-time purchase**.
+- Licenses have **no expiration** (perpetual use is possible).
+- Licenses do not include **support**. If support is needed, please consult separately.
+- Licenses apply to the **latest version** at the time of acquisition. Upgrade fees may apply if there is a major update.
+- If commercial use is confirmed after starting use as personal, educational, or non-profit use, **commercial license fees will be charged from the start of commercial use**.
+- **Exhibitions and demonstrations** aimed at increasing the project's recognition are not subject to commercial licenses.
 
-For commercial license details and purchase information, please contact support@aituberkit.com.
+For details and purchase methods of commercial licenses, please contact support@aituberkit.com.
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)